src/current/v23.1/cdc-queries.md

```diff
@@ -190,6 +190,57 @@ For newly inserted rows in a table, the `cdc_prev` column will emit as `NULL`.
 If you do not need to select specific columns in a table or filter rows from a changefeed, you can instead create a changefeed using the [`diff` option]({% link {{ page.version.version }}/create-changefeed.md %}#diff-opt) to emit a `before` field with each message. This field includes the value of the row before the update was applied.
 {{site.data.alerts.end}}
 
+### Reference TTL in a CDC query
+
+In CockroachDB, table row deletes occur as a result of regular SQL transactions or through [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}). When your changefeed emits [delete event messages]({% link {{ page.version.version }}/changefeed-messages.md %}#delete-messages), you may need to distinguish between these two types of deletion. For example, only emitting messages for row-level TTL deletes from your changefeed.
+
+If you have TTL logic defined with [`ttl_expiration_expression` or `ttl_expire_after`]({% link {{ page.version.version }}/row-level-ttl.md %}#syntax-overview), you can leverage CDC queries to determine whether or not a given row was expired at the time of the changefeed event, including a delete event.
+
+When the table uses the `ttl_expire_after` storage parameter, you can emit rows that were deleted after expiring from the changefeed with syntax similar to:
```

Comment:
structurally I'd prefer to see this section come second after `ttl_expiration_expression`, since we are recommending in the [TTL docs](https://www.cockroachlabs.com/docs/stable/row-level-ttl):

> Starting with CockroachDB v22.2, we recommend that most users use `ttl_expiration_expression` over `ttl_expire_after` for the following reasons:
> - If you use `ttl_expiration_expression`, you can use an existing [TIMESTAMPTZ](https://www.cockroachlabs.com/docs/v23.1/timestamp) column called e.g. updated_at.
> - If you use `ttl_expire_after`, it will cause a full table rewrite, which can affect performance. Also, you can't use it with an existing [TIMESTAMPTZ](https://www.cockroachlabs.com/docs/v23.1/timestamp) column. 

also I think it might be worth even pulling in the recommendation above w.r.t recommending `ttl_expiration_expression` for most users (maybe via include)